cli/flox/src/commands/activate.rs

```diff
@@ -264,6 +266,7 @@ impl Activate {
             ),
         ]);
 
+        let socket_path = environment.services_socket_path(&flox)?;
```

Comment:
Oof, this prevents activations for remote environments. I'll look into it when back at a computer.

Reply:
Fixed in 2400a70. Critique welcome but would prefer to follow-up any fixes in a separate PR.